### PR TITLE
Dev: Add CodeCov report to Travis CI

### LINKED ISSUE #171: Dev: Add CodeCov Report to Travis CI
## Description
As a maintainer,
I need to add CodeCov report to the Travis CI,
so that I can make sure the code quality written in the PRs.

## Mocks
NA

### Update [Required]
- [ ] Update Travis CI to add CodeCov report

### Enhancement to Update [Optional]
NA

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
1 hr

### PR DESCRIPTION
**Pull request by mtreacy002**
### Description
Added CodeCov report to Travis CI

Fixes #171 

### Type of Change:
- Code
- Quality Assurance

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
Pushing this PR to develop showed Travis CI build as pass and generated CodeCov report:
* on CodeCov badge
Screen Shot 2020-10-26 at 12 11 58 am

* on CodeCov.io
Screen Shot 2020-10-26 at 12 19 41 am

### Checklist:
- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -26,5 +26,6 @@ before_script:
 install:
   - pip3 install -r requirements.txt
 script:
-  - python -m unittest discover tests -v
+  - coverage run -m unittest discover tests
   - pytest --cov tests
+  - bash <(curl -s https://codecov.io/bash)
```
Comment (devkapilbansal) [inline .travis.yml]: There should be a blank line at the end

**Review by devkapilbansal — CHANGES_REQUESTED** [review]

**Comment by devkapilbansal** [comment]
@mtreacy002 the commit here is not linked too your github account. Please edit the commit author and make necessary changes

File: .travis.yml (hunk 1 above)
Comment (mtreacy002) [reply .travis.yml]: @devkapilbansal, can you please elaborate why we need this blank line? would like to learn the reason behind it 😉.

**Comment by mtreacy002** [comment]
> @mtreacy002 the commit here is not linked too your github account. Please edit the commit author and make necessary changes

mmm... I'm not sure why you think this is the case, @devkapilbansal . Can you go to my fork repo (which has my GitHub id) from the link stated under the PR title? If you can, isn't that mean this PR is linked to my GitHub account?
Screen Shot 2021-02-19 at 9 28 57 am

File: .travis.yml (hunk 1 above)
Comment (devkapilbansal) [reply .travis.yml]: It is as per PEP8 guidelines `W391`. All python files should have one and only one blank line at the end. 
Actually it's not like adding a blank line at the end but terminating the line with a newline character (although both sounds same)

**Comment by devkapilbansal** [comment]
@mtreacy002 look here
Screenshot_20210219-040452

**Comment by mtreacy002** [comment]
> @mtreacy002 look here
> Screenshot_20210219-040452

I see. Ok. will look into it. Thanks for bringing it up, @devkapilbansal.

**Comment by mtreacy002** [comment]
accidentally closed 🤣🤣

**Pull request closed without merging**